apps/rehabstod/src/components/dialog/Dialog.tsx

```diff
@@ -0,0 +1,59 @@
+import { IDSDialog, IDSDialogElement } from '@frontend/ids-react-ts'
```

Comment:
Tests are missing

Reply:
Added some tests. Attempted to add some tests that trigger events from the shadow DOM, but was unsuccessful.